esp32/modules/appglue.py

```diff
@@ -2,8 +2,15 @@
 
 def start_app(app):
     ugfx.clear(ugfx.WHITE)
-    ugfx.string(0,  0, "Starting "+app+"...","Roboto_Regular12",ugfx.BLACK)
+    ugfx.string(0, 0, "Please wait...", "PermanentMarker22", ugfx.BLACK)
+    if (app==""):
+        ugfx.string(0,  25, "Returning to homescreen...","Roboto_Regular12",ugfx.BLACK)
```

Comment:
as we expect the homescreen to work I'd be OK with skipping this intermediate text+flush and just rebooting immediately